src/main/java/org/schemaspy/SchemaAnalyzer.java

```diff
@@ -76,6 +71,9 @@ public SchemaAnalyzer(SqlService sqlService, DatabaseService databaseService, Co
         this.databaseService = Objects.requireNonNull(databaseService);
         this.commandLineArguments = Objects.requireNonNull(commandLineArguments);
         addOutputProducer(new XmlProducerUsingDOM());
+        if(commandLineArguments.isRenderDotInJvm()){
```

Comment:
Missing else statement, but as I commented earlier maybe move this to Schemaspyconfiguration.